charts/linkerd2/templates/proxy-injector-rbac.yaml

```diff
@@ -83,11 +83,15 @@ metadata:
 webhooks:
 - name: linkerd-proxy-injector.linkerd.io
   namespaceSelector:
+    {{- if .Values.proxyInjector.namespaceSelector }}
+    {{- toYaml .Values.proxyInjector.namespaceSelector | trim | nindent 4 }}
```

Comment:
I did not run the PR yet, but As I can see `matchExpressions` already below, and from [the api](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#labelselector-v1-meta) It seems like there is only one `matchExpression` allowed? :thinking:

Reply:
I'm not sure I follow? It is just a block where the user can override what is set by default? It's no different from how you might plumb in a resources block or a set of custom sidecar containers?